tests/network/hotplug_bridge.go

```diff
@@ -380,8 +387,13 @@ func addBridgeInterface(vm *v1.VirtualMachine, name, netAttachDefName string) er
 }
 
 func verifyBridgeDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, plugMethod hotplugMethod) *v1.VirtualMachineInstance {
+	if plugMethod == migrationBased {
+		migration := libmigration.New(vmi.Name, vmi.Namespace)
+		migrationUID := libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(kubevirt.Client(), migration)
+		libmigration.ConfirmVMIPostMigration(kubevirt.Client(), vmi, migrationUID)
+	}
```

Comment:
Can this part move to the test code as in the un-plug tests (286)?

Reply:
If I extract this portion out to the callers, a very large amount of code will be added.

I guess some more refactoring can be made on these tests, but my current goal was to avoid having shared helpers used between the network tests which are not in `libnet`.
I think that goal has been accomplish and anything beyond that can be done as a follow up.

Most likely, grouping these calls for a migration could fit under `libmigration`.